.github/workflows/Documentation.yaml

```diff
@@ -93,18 +93,24 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      # This is needed to ensure that this workflow can run when called by workflows in external repositories
+      - uses: actions/checkout@v4
+        with:
+          repository: canonical/oci-factory
+          path: oci-factory
```

Comment:
you are missing the `ref`, as suggested in my previous review. Otherwise you're always gonna clone `main`, and for that, you might as well just do 

`        uses: canonical/oci-factory/.github/actions/fetch-releases-json@main`

The point is that, when running in the oci-factory repo, you may want to test this custom action, and thus run it from whatever branch it's being developed